docs/template.html

```diff
@@ -425,6 +425,7 @@
   </style>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta charset="utf-8">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/elasticlunr/0.9.6/elasticlunr.min.js" type="text/javascript"></script>
```

Comment:
I don't want the default template to be tied to this specific library, if you create a repo with a template that adds this feature I'd be very happy to link to it from the README!